src/registry/run.ts

```diff
@@ -0,0 +1,40 @@
+import { type SignerConfig } from "../helpers/connection.js";
+import { type ContractTransaction } from "../helpers/ethers.js";
+import { validateTableName } from "../helpers/parser.js";
+import {
+  getContractAndOverrides,
+  TableIdentifier,
+  assertChainId,
+} from "./contract.js";
+
+export interface PrepareParams {
+  statement: string;
+  chainId: number;
+  first: string;
+}
+
+export async function prepareRunSQL({
```

Comment:
Same pattern as with create. We don't try to parser or catch mistakes here at all (the tests also mention this). That is what the higher level APIs are for!